### ISSUE TITLE
Fen issue with TV shows

### ISSUE THREAD

**Issue by baltzis** [Tikipeter/repository.tikipeter#312]
A couple of days ago, and without making any changes, updates etc., Fen stopped working for TV shows: From a very large Trakt collection of shows, Fen shows only 6 shows in my Trakt collection (the same goes for my trakt watchlist and any other list I use). When trying to make a search (e.g. of known TV shows), it produces no results. When trying to use Fen to play some episode (e.g. using OpenMeta), it produces the error shown in the attached images.

It is noteworthy that until Sunday night, Fen worked great for TV shows and it still works great with movies (and the shows it can "see"). The same issue appears on both my Desktop and TV box, so I tried a fresh Kodi installation on my Laptop, however, the issue persists.

Is this some problem with my installation or is it some Fen bug?

Thanks
![image01](https://user-images.githubusercontent.com/26595356/99521145-b4ce7b80-299c-11eb-8822-30e74af129af.png)
![image02](https://user-images.githubusercontent.com/26595356/99521180-c44dc480-299c-11eb-886b-08a4b2063e0c.png)

**Comment by Tikipeter** [comment]
I need a full log please, not "pictures". Getting it from your desktop install would be easiest.

**Comment by baltzis** [comment]
Thanks for your prompt response. Finally, after several tests, I discovered that the problem is caused when language in Tiki Meta is set to other than English. It was set in Greek, though, for a long time and only since a couple of days ago this problem appeared. After several tests, and on two different devices (laptop and TV box), I confirmed that this is the case. Restoring language in English, in Tiki Meta, solved the problem and Fen works as usual. Perhaps this is some bug.

As I am not familiar with technical details, I do not know how to extract a full log. Before opening this issue, I cleaned all cache and settings. I even installed Kodi on my laptop. If you give me some instructions, I would be glad to help providing the log file.

**Comment by Tikipeter** [comment]
Let me look into it...

**Comment by baltzis** [comment]
Thanks again.
I found the log and I am attaching it. I also found that the problem is partially solved. For some TV shows, Fen still produces the same error and it does not present them in the collection/library listings. Performing a search still does not produce any results.

I updated Fen and cleaned all cache. I cleaned also cache in Tiki Meta and set language in English.

The log I am attaching is from my desktop computer.
[kodi.log](https://github.com/Tikipeter/repository.tikipeter/files/5565056/kodi.log)

**Comment by Tikipeter** [comment]
Can you give me an example of 1 or 2 shows that won't display?

There is an issue here and I will have better luck tracking it down if I know a couple of shows that will make this happen.

**Comment by baltzis** [comment]
For example _Fauda_ or _Senke nad Balkanom_ (also known as _Black Sun_ or _Shadows over Balkans_). I have seen some episodes of both shows using Fen, but they do not appear anymore when checking my listings in Fen, although included in them (Kodi library, Trakt collection). They also do not appear where trying "Search". When trying to show some episode using Fen, the error is produced. Probably there are other shows too, but I did not have the time to test them. I can make more tests during this weekend.

**Comment by baltzis** [comment]
Other examples of shows not appearing in Fen, although included in my lists: 
Babylon Berlin
Britannia
Broadchurch
Death in Paradise
Keeping Faith
Law & Order: True Crime
Taboo
The Last Kingdom
The Road to Calvary

**Comment by Tikipeter** [comment]
Unfortunately I'll be busy now for a few days, but this will be the first thing I look into.

**Comment by Tonkapug** [comment]
No issues here searching! I only attempted #1 & #2 on your list @baltzis

**Comment by baltzis** [comment]
I really can't understand what is going on! It seems that all of a sudden the problem disappeared by itself. I tried several times to repeat it. I tried to isolate it by installing Kodi and only Fen on another computer, to make sure that the language settings in Tiki Meta caused it. I did not manage to make it appear again, neither on this "lab experiment", nor on my TV box. The only issue I face, is that for the same TV series, my Laptop Fen finds several links, but on the TV Box Fen finds none. Both devices are cable connected to the Internet from the same location.

Anyway, thanks for your effort.
